public/app/features/alerting/unified/components/GrafanaAlertmanagerDeliveryWarning.tsx

```diff
@@ -14,12 +14,12 @@ interface GrafanaAlertmanagerDeliveryWarningProps {
 
 export function GrafanaAlertmanagerDeliveryWarning({ currentAlertmanager }: GrafanaAlertmanagerDeliveryWarningProps) {
   const styles = useStyles2(getStyles);
-
-  const { useGetAlertmanagerChoiceStatusQuery } = alertmanagerApi;
-  const { currentData: amChoiceStatus } = useGetAlertmanagerChoiceStatusQuery();
-
   const viewingInternalAM = currentAlertmanager === GRAFANA_RULES_SOURCE_NAME;
 
+  const { currentData: amChoiceStatus } = alertmanagerApi.endpoints.getAlertmanagerChoiceStatus.useQuery(undefined, {
+    skip: !viewingInternalAM,
```

Comment:
This is an optimization to skip fetching the configured alertmanager config when viewing a non-Grafana AM